lld/test/wasm/page-size.s

```diff
@@ -0,0 +1,48 @@
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %s -o %t.o
+# RUN: wasm-ld -no-gc-sections -o %t.wasm %t.o --page-size=1
+# RUN: obj2yaml %t.wasm | FileCheck %s
+
+    .section .data.foo,"",@
+    .globl  foo
+    .hidden  foo
+    .p2align        2
+foo:
+    .int32  0xffffffff
+    .size   foo, 4
+
+    .section .text._start,"",@
+    .globl  _start
+_start:
+    .functype _start () -> (i32)
+    i32.const __wasm_first_page_end
+    end_function
+
+    .section .data.bar,"",@
+    .globl  bar
+    .hidden  bar
+    .p2align        2
+bar:
+    .int32  0x11111111
+    .size   bar, 4
+
+
+# CHECK:  - Type:            MEMORY
+# NEXTLN:   Memories:
```

Comment:
Shouldn't these be `CHECK-NEXT:` .. or is `NEXTLN` something I've not heard of before?

Reply:
This is just me trying to use an idiom from cranelift's filecheck program. Thanks for catching it!